### PR TITLE
21747 sync incorporationApplication of BC, CC, ULC

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#21747

*Description of changes:*
- Additional: refactoring and lint fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: colin-api/src/colin_api/exceptions/__init__.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class GenericException(Exception):
 
     def __init__(self, error, status_code, *args, **kwargs):
         """Return a valid GenericException."""
-        super(GenericException, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```
Comment (severinbeauvais) [inline colin-api/src/colin_api/exceptions/__init__.py:26]: Is the comment on line 25 still valid? Also the others in this file.

File: colin-api/src/colin_api/models/business.py [hunk 2]
```diff
@@ -69,27 +64,19 @@ class CorpStateTypes(Enum):
...
-            TypeCodes.CCC_COMP.value
-        ],
-        LearBusinessTypes.EXTRA_PRO_A.value: [
-            TypeCodes.EXTRA_PRO_A.value
-        ],
+    NUMBERED_CORP_NAME_SUFFIX = {
+        TypeCodes.BCOMP: 'B.C. LTD.',
+        TypeCodes.BC_COMP: 'B.C. LTD.',
+        TypeCodes.ULC_COMP: 'B.C. UNLIMITED LIABILITY COMPANY',
+        TypeCodes.CCC_COMP: 'B.C. COMMUNITY CONTRIBUTION COMPANY LTD.',
```
Comment (severinbeauvais) [inline colin-api/src/colin_api/models/business.py:71]: For what it's worth, the UI doesn't add "LTD" to the name.

In shared repo: https://github.com/bcgov/bcrs-shared-components/blob/83cfe9c8dbe19714567814eb3efd7e08f35faf49/src/components/correct-name/CorrectAmlNumbered.vue#L44

In Create UI: https://github.com/bcgov/business-create-ui/blob/1dee380d0e31eaa53a9fb4392f844ab769f147df/src/components/common/NameRequestInfo.vue#L282

In MBR:
image

**Review by severinbeauvais — APPROVED** [review]
As far as I can tell, this looks like good code and cleanup.

File: colin-api/src/colin_api/models/business.py (hunk 2 above)
Comment (vysakh-menon-aot) [reply colin-api/src/colin_api/models/business.py:71]: This is what we implemented in legal-api. We can check with the business

Comment (severinbeauvais) [reply colin-api/src/colin_api/models/business.py:71]: Yes, please ask Naveen.

Comment (vysakh-menon-aot) [reply colin-api/src/colin_api/models/business.py:71]: I found this in colin
image

**Review by argush3 — APPROVED** [review]

File: colin-api/src/colin_api/models/business.py (hunk 2 above)
Comment (vysakh-menon-aot) [reply colin-api/src/colin_api/models/business.py:71]: @severinbeauvais  Will go with LTD. 
image

Comment (severinbeauvais) [reply colin-api/src/colin_api/models/business.py:71]: I've created ticket https://github.com/bcgov/entity/issues/21894 to fix the UI accordingly.

THIS conversation is resolved.

**Pull request merged**